clang/lib/Lex/PPDirectives.cpp

```diff
@@ -19,7 +19,9 @@
 #include "clang/Basic/Module.h"
 #include "clang/Basic/SourceLocation.h"
 #include "clang/Basic/SourceManager.h"
+#include "clang/Basic/TargetInfo.h"
 #include "clang/Basic/TokenKinds.h"
+#include "clang/Frontend/FrontendOptions.h"
```

Comment:
clangLex cannot depend on clangFrontend due to https://llvm.org/docs/CodingStandards.html#library-layering